### PR TITLE
New version: Anaconda.Anaconda3 version 2024.02-1

### PR DESCRIPTION
**Pull request by einar-prod**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/148650)

### CODE REVIEW

**Comment by stephengillie** [comment]
This package installs and launches normally on a Windows 10 VM.

**Comment by stephengillie** [comment]
This package installs and launches normally on a Windows 10 VM.

**Comment by stephengillie** [comment]
Hi @einar-prod,

This manifest removes Apps and Features entries that are present in previous PR versions. These entries should be added to this version, to maintain version matching, and prevent the 'upgrade always available' situation with this package.

(Automated response - build 876.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 6:02:05 PM: 
> 2024-04-15 18:01:50.683 [FAIL] C:\__w\1\s\external\pkg\src\AppInstallerSharedLib\Yaml.cpp(580)\WindowsPackageManager.dll!00007FFB23CF71BB: (caller: 00007FFB23D30AB2) Exception(1) tid(790) 80071392 The object already exists.

(Automated response - build 876.)

**Comment by stephengillie** [comment]
This package installs and launches normally on a Windows 10 VM.

**Comment by stephengillie** [comment]
The scan phase identiifes all new EXEs and launches them. There were so many errors for 2 files that it seems like maybe they should be dependencies. The first is common - `Microsoft.VCRedist.2015+.x64`:
image

The second is less common, and appears to be `Z-Library.Z-Library`:
image

File: manifests/a/Anaconda/Anaconda3/2024.02-1/Anaconda.Anaconda3.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+UpgradeBehavior: uninstallPrevious
+Commands:
+- conda
+- pip
+- py
+- python3
+ReleaseDate: 2024-02-26
+AppsAndFeaturesEntries:
+- DisplayName: Anaconda3 2024.02-1 (Python 3.11.7 64-bit)
+Installers:
```
Comment (stephengillie) [inline manifests/a/Anaconda/Anaconda3/2024.02-1/Anaconda.Anaconda3.installer.yaml:19]:
```suggestion
Dependencies:
  PackageDependencies:
     - PackageIdentifier: Microsoft.VCRedist.2015+.x64
     - PackageIdentifier: Z-Library.Z-Library
Installers:
```

File: manifests/a/Anaconda/Anaconda3/2024.02-1/Anaconda.Anaconda3.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  SilentWithProgress: /S
+UpgradeBehavior: uninstallPrevious
+Commands:
+- conda
+- pip
+- py
+- python3
+ReleaseDate: 2024-02-26
+AppsAndFeaturesEntries:
+- DisplayName: Anaconda3 2024.02-1 (Python 3.11.7 64-bit)
```
Comment (stephengillie) [inline manifests/a/Anaconda/Anaconda3/2024.02-1/Anaconda.Anaconda3.installer.yaml:18]:
```suggestion
- DisplayName: Anaconda3 2024.02-1 (Python 3.11.7 64-bit)
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @einar-prod,

As I mentioned above, both of these appear to be mentioned in various errors from launching all `exe` files. However, with both as dependencies, only the first type of error dsiappeared, yet the second type remained:

image

The first dependency should probably be added - but what about the second? Both are optional. What's your opinion?

File: manifests/a/Anaconda/Anaconda3/2024.02-1/Anaconda.Anaconda3.installer.yaml (hunk 2 above)
Comment (einar-prod) [reply manifests/a/Anaconda/Anaconda3/2024.02-1/Anaconda.Anaconda3.installer.yaml:18]: Tested on win11 VM and seems like zlib.dll should be included with the install and located in C:\ProgramData\anaconda3\Library\lib, in addition it should be available to install via conda: https://anaconda.org/conda-forge/zlib
I agree vcredist should be added, but suggest not including zlib

**Comment by stephengillie** [comment]
This package installs and launches normally on a Windows 10 VM.

**Comment by stephengillie** [comment]
This package installs and launches normally on a Windows 10 VM.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**